style.css

```diff
@@ -0,0 +1,289 @@
+  position: relative;
+  width: 100%;
+  /*max-width: 85%;*/
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.header:before {
+  content: '';
+   position: absolute;
+   top: 0;
+   left: 0;
+   bottom: 0;
+   right: 0;
+   background: rgba(63, 76, 167, 0.7);
+
+}
+
+.header_navbar {
+  display:flex;
+  padding: 1rem;
+  padding-left: 15vw;
+  align-items: center;
+}
+
+
+.navbar_logo {
+  flex-grow: 3;
+  display: flex;
+  align-items: center;
+}
+
+.navbar_logo_img {
+  width: 3rem;
+  height: 3rem;
+  border-radius: 1.5rem;
+  border: .1rem solid black;
+}
+
+.navbar_logo_text {
+  color: #040236;
+  padding-left: .5rem;
+  font-weight: normal;
+
+}
+.header_title {
+
+  color: #dddddd;
+  font-size: 400%;
+  font-weight: normal;
+  text-transform: uppercase;
+  text-shadow: .2rem .2rem #01010b;
+  align-self: center;
+  padding: 3rem 3rem 0rem;
+}
+
+.header_sub-title {
+  align-self: center;
+  color: #040236;
+  padding: .7rem 0rem 7rem;
+}
+
+.navbar_search_about {
+  flex-grow: 1;
+  display: flex;
+  text-transform: uppercase;
+  color: #040236;
+  text-shadow: .02rem .02rem #020500;
+}
+
+.navbar_about {
+  margin: 0 .8rem;
+}
+
+.navbar_search {
+
+}
+
+.search_icon {
+  padding-left: .4rem;
+}
+
+.main {
+
+  max-width: 85%;
+  display: flex;
+  margin: auto;
+  padding: 2rem 0rem;
+
+}
+
+.main_left {
+
+  display: flex;
+  flex-direction: column;
+  flex-basis: 70%;
+  padding-right: 1rem;
+
+}
+
+.main_right {
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-basis: 30%;
+  padding-left: 1rem;
+
+}
+
+.auth_pub_info {
+  display: flex;
+  align-items: center;
+}
+
+.auth_av {
+  width: 2rem;
+  height: 2rem;
+  border-radius: 1rem;
+  padding: .2rem;
+}
+
+.auth_name {
+  padding: 0rem 1rem 0rem 0rem;
+  color: #4191d4;
+}
+
+.pub_date {
+  color: #a1a1a1;
+  text-decoration: underline;
+}
+
+.left_article {
+  padding: 1.8rem 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.left_quote {
+  padding: .5rem;
+  color: #4161d5;
+  border-left: 1px solid black;
+  margin: 1rem;
+}
+
+.auth_img {
+  width: 5rem;
+  height: 5rem;
+  border-radius: 2.5rem;
+}
+
+.auth_name_right {
+  color: #4191d4;
+}
+
+.social_links {
+
+  display: flex;
+  width: 65%;
+  justify-content: space-around;
+  color: #b3b3b3;
+}
+
+.social_links_desc {
+  color: gray;
+}
+
+.ad {
+  display: flex;
+  flex-direction: column;
+}
+.ad_img {
+  padding: 2rem 0rem 0rem 0rem;
+}
+
+.ad_about {
+  align-self: flex-end;
+  color: #b3b3b3;
+}
+
+.side_sub_heading {
+  align-self: flex-start;
+  padding: 2rem 0rem 0rem 0rem;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100vw;
+  max-width: 100%;
+  margin: auto;
+  border-top: 1px solid gray;
+}
+
+.footer_title {
+  color: #416691;
+  font-variant: small-caps;
+}
+
+.footer_row1 {
+  display: flex;
+  width: 100%;
+  height: 15rem;
+  overflow: auto;
+}
+
+.footer_row1_column {
+  max-width: 25%;
```

Comment:
I think this should just be regular `width` there's some weird stuff happening at larger screen sizes where the images don't go across the full screen